eng/native/set-cmake-path.ps1

```diff
@@ -38,8 +38,8 @@ try {
   if ($version -lt [Version]"3.14.0") {
```

Comment:
I am not sure that it is worth having the distinction between recommended and required. Notice that the minumum required version does not match between the docs and the script.

Would it make sense to simplify this in the next infrastructure rollout and just have 3.16.4 as the minumum required version on Windows?

Reply:
Yes, we should just change this to check for 3.16.4 as the minimum. I’ll add that change to one of my infra rollout PRs